worldengine/draw.py

```diff
@@ -425,8 +395,8 @@ def draw_riversmap_on_file(world, filename):
 
 
 def draw_grayscale_heightmap_on_file(world, filename):
-    img = ImagePixelSetter(world.width, world.height, filename)
-    draw_grayscale_heightmap(world, img)
+    img = PNGWriter.grayscale_from_array(world.elevation['data'], filename, scale_to_range=True)
+    #draw_grayscale_heightmap(world, img)
```

Comment:
PNGWriter can take any numpy-array and write it to a PNG-file. In the case of ```world.elevation['data']``` there is no immediate reason to perform any kind of processing before that. I don't see the need for any kind of change - so ```draw_grayscale_heightmap(world, img)``` could technically be an empty function.

This was not due to performance reasons, just because I wouldn't want to see the heightmap tampered with (since when using it as foundation for a 3D-landscape that would probably lead to visual irregularities).